### ISSUE TITLE
Depreciate project wiki

### ISSUE THREAD

**Issue by BobBorges** [welfare-state-analytics/riksdagen-corpus#389]
We link "documentation" from the readme which links to the project Wiki -- ~~Except for a recently added page, mostly last updates were in 2021. Time to get on top of maintaining that.~~

- [ ] unlink wiki in readme
- [ ] move necessary info to `docs/`
- [ ] add note in wiki that it's not maintained

**Comment by ninpnin** [comment]
Alternatively, we can deprecate the Wiki, and concentrate on other documentation (READMEs, paper, python module docs).

**Comment by MansMeg** [comment]
Yes, I agree with @ninpnin . Maybe focus on the README for now and add some additional markdown docs in the doc folder if needed.
